www/main.js

```diff
@@ -21457,7 +21467,7 @@ function wrapProgram(container, i, fn, data, declaredBlockParams, blockParams, d
     var options = arguments.length <= 1 || arguments[1] === undefined ? {} : arguments[1];
 
     var currentDepths = depths;
-    if (depths && context !== depths[0]) {
+    if (depths && context != depths[0]) {
```

Comment:
main.js is generated from all the dependencies, this seems to be a change on handlebars